src/ensembl/src/shared/components/error-screen/ErrorScreen.scss

```diff
@@ -46,7 +46,7 @@
   font-size: 12px;
```

Comment:
You will find above that the `.generalErrorScreen` class has a height of `100vh`. This is too much for the 404 page when it is rendered client-side, with the top bar and the launchbar. Currently, when the 404 page is server-rendered, it does not have the topbar and the launchbar on the top; so the 100vh works. I suppose we will want to include the topbar and the launchbar into the server-rendered payload, but we don't yet have clarity on that.

Anyhow, just wanted to let you know. Not sure what the proper solution should be; there are several options.

Reply:
Changed `100vh` to `100%` as discussed during code review